CHANGELOG.md

```diff
@@ -302,7 +315,6 @@ Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 - build(deps): bump next from 15.3.4 to 15.5.2 in /react-email-preview [`#8660`](https://github.com/opengovsg/FormSG/pull/8660)
 - fix(deps): bump next from 15.3.4 to 15.5.2 [`#8661`](https://github.com/opengovsg/FormSG/pull/8661)
 - build(deps-dev): bump vite from 5.4.19 to 5.4.20 in /frontend [`#8686`](https://github.com/opengovsg/FormSG/pull/8686)
-- fix: home number fields are added to the draft despite not being modified  [`#8704`](https://github.com/opengovsg/FormSG/pull/8704)
```

Comment:
this change was [deployed to production on 16th Sept](https://github.com/opengovsg/FormSG/pull/8897/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR326), but only merged into develop on 17th Sept.